workflow/transport/create_road_network.py

```diff
@@ -83,22 +85,26 @@ def get_road_condition(row: pd.Series) -> Tuple[str, str]:
         row: Must have surface (nullable) and highway attributes.
 
     Returns:
-        Boolean paved status and surface category string
+        surface category string
     """
-
     if not row.tag_surface:
         if row.tag_highway in {"motorway", "trunk", "primary"}:
-            return True, "asphalt"
+            return "asphalt"
         else:
-            return False, "gravel"
+            return "gravel"
     elif row.tag_surface == "paved":
-        return True, "asphalt"
+        return "asphalt"
     elif row.tag_surface == "unpaved":
-        return False, "gravel"
-    elif row.tag_surface in {"asphalt", "concrete"}:
-        return True, row.tag_surface
+        return "gravel"
```

Comment:
If `tag_surface` isn't set, or it's 'unpaved', I think we should return some null value rather than 'gravel'; we don't know the material. Later, the `get_road_is_paved` function will set anything without a material to `paved = False`.